packages/webview_flutter/webview_flutter_wkwebview/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.4.5
+
+* Adds support for `limitsNavigationsToAppBoundDomains` in `WebKitWebViewControllerCreationParams`
```

Comment:
Preferred style for the `CHANGELOG` is to provide a short explanation of the feature and then state the method. e.g.

```markdown
* Adds support to limit navigation to pages within the app’s domain. See
  `WebKitWebViewControllerCreationParams.limitsNavigationsToAppBoundDomains`.
```